### ISSUE TITLE
Incorret using of heading in Home Page

### ISSUE THREAD

**Issue by brunopulis** [brunopulis/brunopulis.com#226]
**Describe the bug**
Using the incorrect order in Headings

**To Reproduce**
1. Go to [Home Page](https://brunopulis.com)
2. Start the VoiceOver
3. Using the **Web Rotor** to read all heading levels
4. See the error 

**Expected behavior**
All cards should be using a `<h2>` in the title and `<p>` to excerpt.

**Desktop (please complete the following information):**
 - OS: [iOS]
 - Browser [chrome]
 - Version [85]
